packages/react-badge/Spec.md

```diff
@@ -120,14 +120,44 @@ interface BadgeProps extends ComponentProps, React.HTMLAttributes<HTMLElement> {
 
 A Presence Badge represents someone's availbility or status
 
-#### API
+```typescript
+/**
+ * {@docCategory PresenceBadge}
+ */
+export type PresenceBadgeStatus = 'busy' | 'oof' | 'away' | 'available' | 'offline';
+
+/**
+ * {@docCategory PresenceBadge}
+ */
+export interface PresenceBadgeProps extends Omit<BadgeProps, 'shape' | 'appearance'> {
+  /**
+   * A PresenceBadge can represent several status
+   * @defaultvalue available
+   */
+  status?: PresenceBadgeStatus;
+  /**
+   * A PresenceBadge can represent status of someone out of the office
+   * @defaultvalue true
+   */
+  inOffice?: boolean;
```

Comment:
Normally I'm in favor of boolean values that are positively-named, but in this case I think it makes more sense to call this `outOfOffice` (default false) since that's the more commonly-used term.

Also, it's potentially confusing that there's an `outOfOffice` status, as well as an `outOfOffice` prop. Maybe clarify in the comment:
"Modifies the display to indicate that the user is out of office. This can be combined with any status to display an out-of-office version of that status."